### ISSUE TITLE
Missing translation (PL) issues

### ISSUE THREAD

**Issue by SirYaro** [prikhodkop/kdm_story_app#67]
1. random hunt event popup consists two untranslatable strings
2. subtitles before showing hunt page can't be translated

![2_dump_200728233303](https://user-images.githubusercontent.com/5879723/89732528-c6c08a00-da4f-11ea-9ad2-801b26017785.png)

3. Card types groups can't be translated:
![dump_200809150733](https://user-images.githubusercontent.com/5879723/89732936-523b1a80-da52-11ea-8bba-8b62ce196c9d.png)

Apologies if that's only me being bling. I can't find this phrases anywhere.

**Comment by prikhodkop** [comment]
1. Fixed.
2. Hm. Did you try to cope subtitle_texts.js file to your translation? These texts use common text loading mechanism, so i expected translation to function out of the box :/
3. Fixed.

**Comment by prikhodkop** [comment]
Yes, i tested. Localization of subutitles work. You just need to do as always: copy subtutle_texts.js file to localization and update texts there.

Tested with russian version:
![age_1](https://user-images.githubusercontent.com/7735127/90291923-8d41b180-de89-11ea-9c27-78fc2fbc1634.jpg)

**Comment by SirYaro** [comment]
4. Still can't find this:
![1_dump_200728230444](https://user-images.githubusercontent.com/5879723/89732527-c58f5d00-da4f-11ea-8322-e8e2da22d0ac.png)

```
(yaro)➜  lists git:(master) ✗ > pwd              
/www/kdm_story/kdm_story_app/versions/1.5 (Polish)/text/lists
(yaro)➜  lists git:(master) ✗ > grep Submit *js  
(yaro)➜  lists git:(master) ✗ > grep "number" *js
(yaro)➜  lists git:(master) ✗ > 
(yaro)➜  lists git:(master) ✗ > 

```

**Comment by prikhodkop** [comment]
These lines at texts/tooltips_texts.js: 

'hunt_random_event_popup_text': {
   'text': 'Click icon or type hunt event number',
 },

 'padSubmit': {
   'text': 'submit',
 },

I didn't copy them to polish version yet, though. But you can copy to corresponding file manually.

**Comment by SirYaro** [comment]
5.
I found another one:
![obraz](https://user-images.githubusercontent.com/5879723/90888519-b648d180-e3b6-11ea-9cde-3977fc30d055.png)

**Comment by SirYaro** [comment]
6)
It also seems like kdm_story_app/js/ui/settings.js
can't be translated
